pkg/storage/unified/resource/document.go

```diff
@@ -107,10 +107,22 @@ type IndexableDocument struct {
 	// When the resource is managed by an upstream repository
 	Manager *utils.ManagerProperties `json:"manager,omitempty"`
 
+	// indexed only field for faceting manager info
+	ManagedBy string `json:"managedBy,omitempty"`
+
 	// When the manager knows about file paths
 	Source *utils.SourceProperties `json:"source,omitempty"`
 }
 
+func (m *IndexableDocument) UpdateCopyFields() *IndexableDocument {
+	m.TitleNgram = m.Title
+	m.TitlePhrase = strings.ToLower(m.Title) // Lowercase for case-insensitive sorting ?? in the analyzer?
```

Comment:
we also force the title sort value to lower case.  seems this was the only way to get case insensitive sorting